index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
         </ul>
         <div class="illustration"></div>
     </section>
+    <section class="portfolio">
+        <div class="Portfolio_title_container">
+            <h2 class="portfolio_title">My Recent Works</h2>
+        </div>
+        <div class="separation">
+            <hr>
+        </div>
+        <div class="feature-project_preview_image"></div>
+        <div class="feature-project_title">
+            <h3>Architecture</h3>
+        </div>
+        <div class="feature-project_description_container">
+            <h4 class="feature-project_description">A daily selection of privately personalized reads; no 
+            accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</h4>
+        </div>
+        <ul class="feature-project_skills">
+            <li class="feature-skill_container"><h4 class="feature-skill">CSS</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">HTML</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">Bootstrap</h4></li>
+            <li class="feature-skill_container"><h4 class="feature-skill">Ruby</h4></li>
+        </ul>
+            <button class="feature-project_link">
+                <h4 class="feature-link_text">See project</h4>
```

Comment:
Kindly, I suggest you use the heading tag elements only for a heading and sub-heading title. please, if you want to customize the text inside the `button` tag I suggest you use a `span` instead of a `h4` for good semantics and best practices. 😄